### ISSUE TITLE
Merge reports when running tests in parallel in CircleCI

### ISSUE THREAD

**Issue by samlucax** [cypress-io/cypress#6272]
### Current behavior:

When we run tests in parallel in the CircleCI, the reports are generated for each node.

Still using cypress orb.

### Desired behavior:

Should be able to generate a single report merging all test results from each node.

### Test code to reproduce

``` yml
version: 2.1
orbs:
  cypress: cypress-io/cypress@1
workflows:
  build:
    jobs:
      - cypress/install:
          build: 'npm run build'
      - cypress/run:
          command: "npx cypress run --record --group 'all tests' --parallel"
          requires:
            - cypress/install
          start: 'npm start -- --configuration=staging'
          wait-on: 'http://localhost:4200'
          record: true
          parallel: true
          parallelism: 6
          group: 'all tests'
          post-steps:
            - store_test_results:
                path: cypress/reports
            - store_artifacts:
                path: mochawesome-report
          - run: npm run after:tests
```

#### The output is 

![Screen Shot 2020-01-29 at 19 28 35](https://user-images.githubusercontent.com/11819736/73403154-9bedc000-42cd-11ea-8221-4302a746d5e6.png)

**Comment by jennifer-shehane** [comment]
Duplicate of https://github.com/cypress-io/cypress/issues/1946
